app/javascript/spec/ansible-repository-form/helpers.spec.js

```diff
@@ -0,0 +1,78 @@
+import { urlValidation } from '../../components/ansible-repository-form/helpers';
+
+const TESTCASES = [ // Example Cases
+  [{ scm_url: 'git@github.com:IonicaBizau/node-git-url-parse.git' }, true],
+  [{ scm_url: 'https://github.com/IonicaBizau/node-git-url-parse.git' }, true],
```

Comment:
Let's also add positive cases for the following:

```suggestion
  [{ scm_url: 'https://github.com/IonicaBizau/node-git-url-parse.git' }, true],
  [{ scm_url: 'ssh://github.com/IonicaBizau/node-git-url-parse.git' }, true],
  [{ scm_url: 'file:///opt/repos/IonicaBizau/node-git-url-parse.git' }, true],
```